packages/devtools_app/test/vm_developer/object_inspector/object_inspector_view_test.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/devtools_app.dart';
+import 'package:devtools_app/src/screens/debugger/program_explorer.dart';
+import 'package:devtools_app/src/screens/vm_developer/object_inspector_view.dart';
+import 'package:devtools_app/src/screens/vm_developer/object_viewport.dart';
+import 'package:devtools_test/devtools_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+void main() {
+  final objectInspector = ObjectInspectorView();
+
+  final fakeServiceManager = FakeServiceManager();
+  when(fakeServiceManager.connectedApp!.isProfileBuildNow).thenReturn(false);
+  when(fakeServiceManager.connectedApp!.isDartWebAppNow).thenReturn(false);
+  setGlobal(ServiceConnectionManager, fakeServiceManager);
+  setGlobal(IdeTheme, IdeTheme());
```

Comment:
put all this into a `setUp` method instead. You'll need to define `objectInspector` and `fakeServiceManager` as late outside of the `setUp` method so that they can be accessed everywhere, but they should be initialized in `setUp`